SSD/TestShell/ssdTestShell.cpp

```diff
@@ -85,6 +85,7 @@ class TestShell {
 			cmd += " ";
 			cmd += readedData[1];
 			myExecuter->runner(cmd);
+			cmd = "W ";
```

Comment:
아하 cmd가 for를 돌면서 계속 append가 돘군요
이러면 아에 std::string cmd = "W " 로직에 for문 안에 있게 하면 더 좋을거 같습니다
그리고 간단한 string의 조합이니

```suggestion
			std::string cmd = "W " + to_string(index) + " " + readedData[1];
```

이렇게 한개의 line으로 구현해주셔도 좋을거 같습니다!

Reply:
넵 수정하였습니다. test 후에 제대로 동작하는 것 까지 확인하였습니다.